File: Phyton/Vetor/negativos.py
n = int(input("Quantos numeros voce vai digitar? "))
vet = [0 for x in range(n)]

for i in range(n):
    vet[i] = int(input("Digite um numero: "))


print("Numeros Negativos: ")
for i  in range(n):
    if vet[i] < 0:
        print(vet[i])

File: C/Vetor/Negativos.c
#include <stdio.h>

int main(){
    int i, n;

    printf("Quantos numeros voce vai digitar?");
    scanf("%d", &n);

    int vet[n];
        for(i=0; i<n; i++){
            printf("Digite um numero:");
            scanf("%d", &vet[i]);
            }
            printf("\nNumeros Negativos:\n");
        for (i=0; i<n; i++){
            if(vet[i]<0){
                printf("%d\n", vet[i]);
            }
        }

return 0;
}
